addons/hr_timesheet/static/tests/timesheet_uom_tests.js

```diff
@@ -0,0 +1,94 @@
+/** @odoo-module */
+
+import { session } from "@web/session";
+
+import { makeView } from "@web/../tests/views/helpers";
+import { getFixture, patchWithCleanup } from "@web/../tests/helpers/utils";
+
+import { getServerData, updateArch, setupTestEnv } from "./hr_timesheet_common_tests";
+
+
+QUnit.module("hr_timesheet", (hooks) => {
+    let target;
+    let serverData;
+    hooks.beforeEach(async function (assert) {
+        setupTestEnv();
+        serverData = getServerData();
+        updateArch(serverData, { "unit_amount": "timesheet_uom" });
```

Comment:
No need to convert to key in string like we did in Python. 🙂 
```suggestion
        updateArch(serverData, { unit_amount: "timesheet_uom" });
```